api/src/test/resources/features/callCustomFunction.json

```diff
@@ -0,0 +1,46 @@
+{
+	"document": {
```

Comment:
Instead, can we do 

```yaml
document:
  dsl: 1.0.0-alpha5
  namespace: test
  name: call-example
  version: 0.1.0
schedule:
  cron: 0 8 * * *
do:
- getData:
    call: http
    with:
      method: get
      endpoint: https://api.agify.io?name=meelad
  output:
    as: ".data.reading"
- filterData:
    for:
      in: ".data.reading"
      each: reading
    do:
    - log:
        call: https://raw.githubusercontent.com/serverlessworkflow/catalog/main/functions/log/1.0.0/function.yaml
        with:
          level: information
          format: "{TIMESTAMP} [{LEVEL}] ({CONTEXT}): {MESSAGE}"
          message: Hello, world!
          timestamp: true
```

Reply:
This is intentionally json to have at least one test for json format

Reply:
but probably we should have a different legacy dir for that, so changing